service/rcv_server.conf

```diff
@@ -1,6 +1,19 @@
 [rcv_server]
-roborio_ip=10.0.0.29
-roborio_port=12345
+roborio_ip=10.0.0.25
+roborio_port=5801
+socket_timeout=1
+camera_name=front
+width=320
+height=240
+fps=30
+video_source=video:/home/bazulay/dev_robotics/rcv_videos/edited_videos/2019_02_26_19_39_30.mp4
+focus_image=/opt/rcv/rcv/resources/robot_focus.jpg
+frame_handler=frc_2019
+frame_handlers_dir=frame_handlers
+show=True
```

Comment:
even when turned to false it still found it as true I will put a comment where you should change

Reply:
What do you mean?